src/Native/Unix/System.Native/pal_networking.cpp

```diff
@@ -2081,26 +2081,26 @@ extern "C" Error SystemNative_GetSockOpt(
 
             if (err != 0)
             {
-                return SystemNative_ConvertErrorPlatformToPal(errno);
+                return static_cast<Error>(SystemNative_ConvertErrorPlatformToPal(errno));
```

Comment:
These `static_cast<Error` casts seem to be pretty frequent. Would it make sense to have `ConvertErrorPlatformToPal` local non-exporter variant (without the SystemNative_ prefix) that just returns `Error`?

Reply:
They will all go away as soon as the transition is done, as we won't be able to return `Error` to managed anymore, and just `int32_t`